### PR TITLE
Feature: Reading Urls from file

### PR DESCRIPTION
**Pull request by chr1st1ank**
Firstly, thanks for this great little tool. It works like a charm and is just what I needed.

I was missing one feature and I'm suggesting it here, please let me know if it is wanted. For now, I've implemented and used it only for myself.

The feature needed was, that the load can be distributed over multiple valid URLs. With the regex feature I would have seen almost only HTTP 404 on the service that I wanted to test. So the idea was to get URLs from a real access log and put them into a file. 

This implementation here makes oha:
- take the filename instead of a single URL
- read all the lines and validate if they are valid URLs
- Use random URLs from the list during the load test

The feature is behind the new command line flag `--urls-from-file`.

What do you think about this?

### CODE REVIEW

File: src/url_generator.rs [hunk 1]
```diff
@@ -17,20 +22,43 @@ pub enum UrlGeneratorError {
...
+
     pub fn new_dynamic(regex: Regex) -> Self {
         Self::Dynamic(regex)
     }
 
     pub fn generate<R: Rng>(&self, rng: &mut R) -> Result<Cow<Url>, UrlGeneratorError> {
         match self {
             Self::Static(url) => Ok(Cow::Borrowed(url)),
+            Self::MultiStatic(urls) => {
+                if let Some(random_url) = urls.choose(&mut rand::thread_rng()) {
```
Comment (hatoo) [inline src/url_generator.rs:56]: Please use `rng` argument instead of `thread_rng()`

**Review by hatoo — CHANGES_REQUESTED** [review]
Thank you!
It looks good!

I added a small change request.

I'll merge it after your fix.

File: src/url_generator.rs (hunk 1 above)
Comment (chr1st1ank) [reply src/url_generator.rs]: yes, makes sense, done.

**Comment by chr1st1ank** [comment]
> Thank you! It looks good!
> 
> I added a small change request.
> 
> I'll merge it after your fix.

Ok, great. I went over the code again to improve the error handling and add some tests. Now it's ready.

**Comment by hatoo** [comment]
Thank you!

**Pull request merged**